### PR TITLE
Gulp

### PR DESCRIPTION
**Pull request by normanjaeckel**
We still have to fix three files mentioned in the gulp task.

### CODE REVIEW

**Comment by normanjaeckel** [comment]
See also #1976.

File: gulpfile.js [hunk 1]
```diff
@@ -123,7 +129,13 @@ gulp.task('pot', function () {
 
 // Checks JavaScript using JSHint
 gulp.task('jshint', function () {
-    return gulp.src([ 'gulpfile.js', path.join( 'openslides', '*', 'static', '**', '*.js' ) ])
+    return gulp.src([
+            'gulpfile.js',
+            path.join( 'openslides', '*', 'static', '**', '*.js' ),
+            '!' + path.join( 'openslides', 'users', 'static', 'js', 'users', 'site.js' ),
+            '!' + path.join( 'openslides', 'motions', 'static', 'js', 'motions', 'base.js' ),
+            '!' + path.join( 'openslides', 'motions', 'static', 'js', 'motions', 'site.js' ),
```
Comment (emanuelschuetze) [inline gulpfile.js]: Why do you define these 3 lines for custom apps?

**Comment by emanuelschuetze** [comment]
One remark. Rest is ok.

File: gulpfile.js (hunk 1 above)
Comment (normanjaeckel) [inline gulpfile.js]: These three files are not fixed yet. Someone has to do this and then remove these lines.

Comment (normanjaeckel) [inline gulpfile.js]: You can try it: Remove the lines and run `gulp jshint`.

**Comment by normanjaeckel** [comment]
You can merge it if you don't want to fix the three files at the moment.

File: gulpfile.js (hunk 1 above)
Comment (emanuelschuetze) [inline gulpfile.js]: Ok. Please create an issue. Then you can merge this PR.

Comment (normanjaeckel) [inline gulpfile.js]: See #1979.

**Pull request merged**